CHANGELOG.md

```diff
@@ -30,6 +30,7 @@
 - Accept spans needed for the mobile Starfish module. ([#2570](https://github.com/getsentry/relay/pull/2570))
 - Extract size metrics and blocking status tag for resource spans. ([#2578](https://github.com/getsentry/relay/pull/2578))
 - Add a setting to rollout ingesting all resource spans. ([#2586](https://github.com/getsentry/relay/pull/2586))
+- Add User Feedback Ingestion. ([#2604](https://github.com/getsentry/relay/pull/2604))
```

Comment:
Please also update py/changelog

Reply:
Also, please move this to the `## Unreleased` section at the top of this file.